bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/local/json/LocalDataTypeValue.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.tellstick.internal.local.json;
+
+import org.openhab.binding.tellstick.internal.live.xml.LiveDataType;
+
+/**
+ * Class used to deserialize JSON from Telldus local API.
+ *
+ * @author Jan Gustafsson - Initial contribution
+ */
+public class LocalDataTypeValue {
```

Comment:
You could move this into a package called `dto` or append `DTO` to the class name to get rid of the checkstyle warning about missing NonNullByDefault annotation.

Reply:
Fixed!